### DISCUSSION TITLE
Project Content

### DISCUSSION THREAD

**Discussion by Mankanit** [vkbo/novelWriter#1683] [Help] [answered]
If I have a novel i a simple text editor and want to move it into NovelWriter; is there a way to put marks in the text so parts of the text will show up in the left area Project Content under different scenes or chapters?

**Comment by HeyMyian** [comment, marked as answer]
You can put chapter and scene headings into your text file like this: https://novelwriter.io/docs/usage_format.html#headings

When you load the complete file into one document in novelWriter, you can then in Project Content right click on the file and select Transform > Split Document by Headers.
